accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/installhook/impl/OsgiAwareInstallHook.java

```diff
@@ -45,22 +47,45 @@ public OsgiAwareInstallHook() throws ClassCastException {
 		}
 	}
 
+	// TODO: make protected
```

Comment:
Can we not just make those changes now if they are needed? (or let's don't add todos but rather just create an issue `Make OsgiAwareInstallHook more restrictive`)

Reply:
This requires a major version increase, I don’t want to do right now. I will create an issue though

Reply:
Done in https://github.com/Netcentric/accesscontroltool/issues/394.